### PR TITLE
cmd: remove todos

### PR DESCRIPTION
**Pull request by xenowits**
Removes TODOs in the `cmd` package.

category: misc
ticket: none

### CODE REVIEW

File: cluster/manifest/helpers.go [hunk 1]
```diff
@@ -131,8 +131,6 @@ func SignK1(m *manifestpb.Mutation, secret *k1.PrivateKey) (*manifestpb.SignedMu
 }
 
 // verifyK1SignedMutation verifies that the signed mutation is signed by a k1 key.
-//
-// TODO(corver): Figure out no-verify case.
```
Comment (xenowits) [inline cluster/manifest/helpers.go:135]: We verify k1 signatures, so we don't need a "no-verify" case

File: cmd/addvalidators_internal_test.go [hunk 2]
```diff
@@ -104,7 +104,6 @@ func TestValidateConfigAddValidators(t *testing.T) {
 	}
 }
 
-// TODO(xenowits): Add more extensive tests, this is just a very simple unit test.
```
Comment (xenowits) [inline cmd/addvalidators_internal_test.go:107]: The test is already comprehensive enough to test the "add validators" feature

In addition to that, the add validators feature is not in production, it is under the `charon alpha` command

File: cmd/createdkg.go [hunk 3]
```diff
@@ -77,7 +77,6 @@ func mustMarkFlagRequired(cmd *cobra.Command, flag string) {
 
 func runCreateDKG(ctx context.Context, conf createDKGConfig) (err error) {
 	// Map prater to goerli to ensure backwards compatibility with older cluster definitions.
-	// TODO(xenowits): Remove the mapping later.
```
Comment (xenowits) [inline cmd/createdkg.go:80]: We can remove code for goerli entirely when goerli gets deprecated in a couple of weeks

https://blog.ethereum.org/2024/01/10/goerli-dencun-announcement

File: cmd/relay/relay.go [hunk 4]
```diff
@@ -41,7 +41,7 @@ type Config struct {
 	MaxResPerPeer   int
 	MaxConns        int
 	FilterPrivAddrs bool
-	RelayLogLevel   string // TODO(corver): Rename to LibP2PLogLevel.
```
Comment (xenowits) [inline cmd/relay/relay.go:44]: Renamed ✅

File: cmd/run.go [hunk 5]
```diff
@@ -94,7 +94,6 @@ func bindRunFlags(cmd *cobra.Command, config *app.Config) {
 	})
 }
 
-// TODO(dhruv): add more test only flags to this function.
```
Comment (xenowits) [inline cmd/run.go:97]: We only have one unsafe command for now. We can always add new unsafe commands, no need for a todo for that

**Review by pinebit — APPROVED** [review]

**Pull request merged**